src/locales/en.ts

```diff
@@ -515,6 +515,10 @@ export default {
     memo: {
       label: 'Memo',
       placeholder: 'Enter Memo... (optional)',
+      bytes_left: '{{count}} byte left',
+      bytes_left_plural: '{{count}} bytes left',
+      length_error:
+        'Memo is too long. Please edit the memo to 8 bytes or less.',
```

Comment:
Do you think "bytes" is a concept users will understand?

Reply:
Good question. So it shows the number of bytes you are using as you type. The reason we cant say "8 characters" or something is because emojis or some other language characters are more than 1 byte.

I think with it showing how many bytes you are using as you type you kind of get it even if you don't know what bytes are. Any idea on a way we could make it better?